### PR TITLE
128-improve-translator-interface

### PR DESCRIPTION
**Pull request by klavinsjanis**

### CODE REVIEW

File: pkg/server/message.go [hunk 1]
```diff
@@ -78,13 +81,31 @@ func (t *TranslateServiceServer) CreateMessages(
...
 		if err != nil {
-			return nil, status.Errorf(codes.Unknown, err.Error()) // TODO: For now we don't know the cause of the error.
+			return nil, status.Errorf(codes.Internal, "")
+		}
+
+		for _, v := range msgs {
+			if v.Original {
+				originalLanguage = &v.Language
+			}
+		}
```
Comment (VladislavsPerkanuks) [inline pkg/server/message.go:98]:
```suggestion
		for _, v := range msgs {
			if v.Original {
				originalLanguage = &v.Language
        break
			}
		}
```
Original is only one (hopefully), so no need to continue iterating

File: pkg/fuzzy/aws_translate.go [hunk 2]
```diff
@@ -110,16 +115,6 @@ func (a *AWSTranslate) Translate(ctx context.Context, messages *model.Messages)
 		return &model.Messages{Language: messages.Language, Original: messages.Original}, nil
```
Comment (janishorsts) [inline pkg/fuzzy/aws_translate.go:115]:
```suggestion
		return &model.Messages{Language: targetLanguage, Original: messages.Original}, nil
```

File: pkg/fuzzy/aws_translate.go [hunk 3]
```diff
@@ -5,6 +5,8 @@ import (
 	"fmt"
 	"net/http"
 
+	"golang.org/x/text/language"
+
```
Comment (janishorsts) [inline pkg/fuzzy/aws_translate.go:9]:
```suggestion
	"golang.org/x/text/language"
```

File: pkg/fuzzy/aws_translate.go [hunk 4]
```diff
@@ -112,15 +117,9 @@ func (a *AWSTranslate) Translate(ctx context.Context, messages *model.Messages)
 	// skip locale part if region is not a country,
 	// AWS only supports ISO 3166 2-digit country codes.
 	// https://docs.aws.amazon.com/translate/latest/dg/what-is-languages.html
-	targetLanguage := messages.Language.String()
```
Comment (janishorsts) [inline pkg/fuzzy/aws_translate.go:115]: You removed the code and left the comment that applies to the removed code. Why? :)

File: pkg/fuzzy/fuzzy.go [hunk 5]
```diff
@@ -5,6 +5,8 @@ import (
 	"fmt"
 	"strings"
 
+	"golang.org/x/text/language"
+
```
Comment (janishorsts) [inline pkg/fuzzy/fuzzy.go:9]: Please remove the empty line. Where is the empty line coming from? Do you add imports manually?

**Review by janishorsts — APPROVED** [review]
Please remove the leftover comment and it's good.

**Pull request merged**